express/tests/constants.ts

```diff
@@ -97,7 +97,8 @@ export const TEST_SERVICE_FROM_DYNAMO: ServiceFromDynamo = {
     service_name: {S: TEST_SERVICE_NAME}
 };
 export const TEST_SELF_SERVICE_CLIENT_ID = "someSelfServiceClientId";
-export const TEST_PUBLIC_KEY = "somePubKey";
+export const TEST_PUBLIC_KEY =
+    "MIGfMA0GCSqGSIb3DQEBAQUAA4GNADCBiQKBgQCeQ90Vm6wnW9AYqRTj4bHiFLHWHe1w1u6CaQnkWYeL1q5LCyWofFb0hNZt7h70ZKDvmyDy3Rfw0qwGY9P49neZQJEkSj/VLGrqWhHJEq9Bi4xKxtTo2as8c+uNP8uede6fRTGIO5miaaiR4dhHdViWE2v8LDx0vWkBAi30Ry3kowIDAQAB";
```

Comment:
was `somePubKey` hitting some validation issues somewhere?

Reply:
Yeah the key validation was _rightly_ throwing at the prospect of  `somePubKey`